_assets/scripts/create_vm.py

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env python3
+# Create a VM on a diskless flavor
+# (c) Kurt Garloff <garloff@osb-alliance.de>, 2023-08-20
+# SPDX-License-Identifier: Apache-2.0
+
+import openstack
+# import os
+import sys
+
+# OS_CLOUD will be taken from environment
+FLAVOR="SCS-1V-2"
+NETWORK=""
+IMAGE="Debian 12"
+KEYNAME="SSHkey-gxscscapi"
+
+def main(argv):
+	"""creates a server on a diskless flavor in a cloud env["OS_CLOUD"]
+	   (configured in clouds.yaml+secure.yaml) with flavor FLAVOR
+	   in network NETWORK (defaults to first if empty) with image IMAGE
+	   injecting keypari KEYNAME."""
```

Comment:
This is just a personal preference and more in line with https://peps.python.org/pep-0257/#multi-line-docstrings

```suggestion
	"""
	Creates a server on a diskless flavor in a cloud env["OS_CLOUD"]
	(configured in clouds.yaml+secure.yaml) with flavor FLAVOR
	in network NETWORK (defaults to first if empty) with image IMAGE
	injecting keypair KEYNAME.
	"""
```